### PR TITLE
Use more Android KTX Kotlin extensions 

### PR DESCRIPTION
**Pull request by mcarare**
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt [hunk 1]
```diff
@@ -25,7 +24,7 @@ import androidx.annotation.RequiresApi
 import androidx.browser.customtabs.CustomTabColorSchemeParams
 import androidx.browser.customtabs.CustomTabsIntent
 import androidx.browser.customtabs.CustomTabsIntent.SHARE_STATE_ON
-import androidx.core.content.getSystemService
```
Comment (mcarare) [inline app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt:28]: this was an unused import that got "optimized" 😁

File: app/src/main/java/org/mozilla/focus/appreview/AppReviewUtils.kt [hunk 2]
```diff
@@ -100,15 +100,20 @@ class AppReviewUtils {
             )
 
             return inAppReviewStep == AppReviewStep.ReviewNeeded.name || (
-                lastReviewedTime +
-                    APP_REVIEW_TIME_TRIGGER <= System.currentTimeMillis() &&
-                    inAppReviewStep == AppReviewStep.Reviewed.name
-                )
+                    lastReviewedTime +
```
Comment (mcarare) [inline app/src/main/java/org/mozilla/focus/appreview/AppReviewUtils.kt]: weird, I haven't changed the indentation here, and locally the dif does not show this.

**Review by jonalmeida — APPROVED** [review]
LGTM - I didn't see anything that might be suspicious to avoid landing today but maybe it's best to leave it to Monday? Will leave it up to you to decide. I have no strong feeling in this case.

**Comment by mcarare** [comment]
Monday it is! 😄

**Pull request merged**